modules/nodes-scheduler-node-selectors-cluster.adoc

```diff
@@ -69,35 +128,120 @@ spec:
 +
 After making this change, wait for the pods in the `openshift-kube-apiserver` project to redeploy. This can take several minutes. The default cluster node selector does not take effect until the pods redeploy.
 
-. Edit a node or MachineSet to add labels:
+. Add labels to a node using a MachineSet or editing the node directly:
+
+* Use a MachineSet to add labels to nodes managed by the MachineSet when a node is created:
+
+.. Run the following command to add a node selector to a MachineSet:
 +
 [source,terminal]
 ----
-$ oc label <resource> <name> <key>=<value>
+$ oc patch MachineSet <name> --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"<key>"="<value>","<key>"="<value>"}}]'  -n openshift-machine-api
 ----
 +
-For example, to label a node:
+For example:
 +
 [source,terminal]
 ----
-$ oc label nodes ip-10-0-142-25.ec2.internal type=user-node region=east
+$ oc patch MachineSet ci-ln-l8nry52-f76d1-hl7m7-worker-c --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"type":"user-node","region":"east"}}]'  -n openshift-machine-api
 ----
 +
-To label a MachineSet:
+.. Verify that the label is added to the MachineSet using the `oc edit` command:
++
+For example:
 +
 [source,terminal]
 ----
-$ oc label MachineSet abc612-msrtw-worker-us-east-1c type=user-node region=east
+$ oc edit MachineSet ci-ln-l8nry52-f76d1-hl7m7-worker-c -n openshift-machine-api
 ----
-
-When you create a pod, {product-title} adds the appropriate `<key>:<value>` and schedules
-the pod on the labeled node.
-
++
 .Example output
 [source,yaml]
 ----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+...
 spec:
-  nodeSelector:
-    region: east
-    type: user-node
+...
+  template:
+    metadata:
+...
+    spec:
+      metadata:
+        labels:
+          region: east
+          type: user-node
+----
+
+.. Redeploy the nodes associated with that MachineSet:
++
+For example:
++
+[source,terminal]
+----
+$ oc scale --replicas=0 MachineSet ci-ln-l8nry52-f76d1-hl7m7-worker-c -n openshift-machine-api
+----
++
+[source,terminal]
```

Comment:
I am not sure why we're running this command twice. I suggest adding a little more context to line 177.

Reply:
Scale down to 0 to deschedule the nodes, then scale up to redeploy the node with the new label.

Reply:
I suggest separating the commands into different substeps with that explanation - I worry that people might either miss the second command or just not do one of them, then wonder what went wrong.